suites/quincy/rbd/tier-3_rbd_persistent_write_back_cache.yaml

```diff
@@ -410,3 +411,28 @@ tests:
           desc: RBD Persistent Cache invalidate image level
           module: test_rbd_persistent_writeback_cache_invalidate.py
           name: PWL cache path validation at image level
+
+  - test:
+      abort-on-fail: true
+      config:
+        level: client                        # PWL at client
+        cache_file_size: 1073741824          # 1 GB
+        rbd_persistent_cache_mode: ssd       # "ssd" or "rwl" on pmem device
+        client: node5
+        drive: /dev/nvme0n1
+        cleanup: true
+        validate_exclusive_lock: true
+        rep-pool-only: True
+        rep_pool_config:
+          pool: pool1
+          image: image1
+          size: 10G
+        fio:
+          image_name: image1
+          pool_name: pool1
+          runtime: 120
+      desc: Validate concurrent writes to exclusive lock feature with persistent cache enabled
+      destroy-cluster: false
+      module: test_concurrent_writes_rbd_persistent_writeback_cache.py
```

Comment:
module name is very big, it's good to keep small e.g
`test_concurrent_write_pwl_cache.py`

Reply:
done.